roles/serf/files/serf.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Serf
-After=auditd.service systemd-user-sessions.service time-sync.target
+After=network.target auditd.service systemd-user-sessions.service time-sync.target
```

Comment:
hmm, that's a bummer. Thanks for checking @vvb 

May be this command doesn't return with non-zero code(I am not able to validate this as I can't seem to disable iptables in a running system) or may be when firewalld comes up it wipes out existing rules (this seems more likely). 

I think what you have works then.